apps/extension/src/App/Accounts/AddAccount.tsx

```diff
@@ -112,8 +112,8 @@ const AddAccount: React.FC<Props> = ({ accounts, requester, setAccounts }) => {
     e: React.ChangeEvent<HTMLInputElement>,
     callback: (value: number) => void
   ): void => {
-    const result = e.target.value.replace(/\D/g, "") || "0";
-    callback(parseInt(result));
+    const result = e.target.value || "0";
+    callback(parseInt(result, 10));
```

Comment:
nit: what if this fails? should we handle it in a graceful way?

Reply:
I'm not sure it could fail - the input comes from `<input type="number" />` which should only allow numerical input. The value is returned as a string, though, so it needs to be parsed, but I'm not sure it's possible to trigger a failure in this case.